### ISSUE TITLE
只要运行Main.py就行？请给下具体操作流程，非常感谢！！！

### ISSUE THREAD

**Issue by xm1112** [KOD-Chen/YOLOv2-Tensorflow#1]

**Comment by lovekittynine** [comment]
你好，请问下，为什么在训练过程中，batchnorm 参数training=Flase,不应该为True吗？

**Comment by Cecilia520** [comment]
我运行了Main.py,没有什么作用呀
